<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure AppCafe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SoftwareManager.qrc">
    <normaloff>:/icons/appcafe.png</normaloff>:/icons/appcafe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Repository Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Note: It may take a couple minutes to apply changes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radio_production">
            <property name="text">
             <string>Production (Updates Quarterly)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_edge">
            <property name="text">
             <string>Edge (Updates Frequently)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_custom">
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="group_custom">
            <property name="title">
             <string>Custom Repositories (only 1 can be active)</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QToolButton" name="tool_addRepo">
                 <property name="toolTip">
                  <string>Add new custom repository</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../pc-softwaremanager/SoftwareManager.qrc">
                   <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="tool_remRepo">
                 <property name="toolTip">
                  <string>Remove currently selected repository</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../pc-softwaremanager/SoftwareManager.qrc">
                   <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Remote Access</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Allow remote access to AppCafe? This allows you to manage packages and jails from a web-browser or mobile device.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAppCafe">
         <property name="title">
          <string>Enable AppCafe Remote</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineUser">
            <property name="placeholderText">
             <string>New Username</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="linePass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>New Password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Repeat:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="linePass2">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>New Password (repeat)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinPort">
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>8885</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <spacer name="verticalSpacer_28">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>59</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="SoftwareManager.qrc">
         <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_apply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="SoftwareManager.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SoftwareManager.qrc"/>
  <include location="../pc-softwaremanager/SoftwareManager.qrc"/>
 </resources>
 <connections/>
</ui>
